src/components/calcite-shell-panel/calcite-shell-panel.scss

```diff
@@ -80,17 +67,12 @@
   --calcite-shell-panel-detached-max-height: 80vh;
 }
 
-.content--detached {
-  @apply rounded
-  shadow-0
-  h-auto
-  overflow-hidden
-  mt-2 mb-auto mx-2;
-
+.content.content--detached {
```

Comment:
I think it should just need `.content--detached {` right?

Reply:
I feel like I had to do it that way...but I'll review.